data/web/default_panels.json

```diff
@@ -0,0 +1,17 @@
+{
+    "0": { "id": "scene", "name": "Scene", "visible": true, "enabled": true },
+    "1": { "id": "settings", "name": "Settings", "visible": false, "enabled": true },
+    "2": { "id": "navigation", "name": "Navigation", "visible": true, "enabled": true },
+    "3": { "id": "timePanel", "name": "Date & Time", "visible": true, "enabled": true },
+    "4": { "id": "sessionRecording", "name": "Session Recording", "visible": false, "enabled": true },
+    "5": { "id": "geoLocation", "name": "Geo Location", "visible": false, "enabled": true },
+    "6": { "id": "screenSpaceRenderables", "name": "ScreenSpace Renderables", "visible": false, "enabled": true },
+    "7": { "id": "exoplanets", "name": "Exoplanets", "visible": false, "enabled": true },
+    "8": { "id": "userPanels", "name": "User Panels", "visible": false, "enabled": true },
+    "9": { "id": "actions", "name": "Actions", "visible": false, "enabled": true },
+    "10": { "id": "skyBrowser", "name": "SkyBrowser", "visible": false, "enabled": true },
```

Comment:
All of these were ones that we had visible per default before. Should they still be? 

At least actions, session recording, screen space renderables?

If not, should we update some of the profiles? E.g. to show the skybrowser and exoplanets in default?  Maybe we don't need it?

Having some conflicting thoughts when writing this, so sorry if it's hard to understand :)

Reply:
I changed their values to mimic the behavior of the previous UI.   In case we have the time to make a full pass through the profiles to adjust, we can change it back again

Reply:
Maybe there is some confusion regarding enabled and visible. Enabled doesn't really do much anymore. We've had some discussions about removing certain panels when we don't want to open them (exoplanets and skybrowser) but I think we've landed on that we want to enable all panels always, but that some are visible. 

Visible on the other hand, is if it is shown in the bottom bar.

Reply:
I updated the file between the comments.. A bunch of the things were accidentally actually disabled, but I updated the file since to represent what our default value are